Housing ML Book.ipynb

```diff
@@ -517,7 +517,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
```

Comment:
Sorry for this mess, it seems that the execution_count changed by 1 for each cell, because I inserted a new cell. Didn't find out how to handle this.